openhtf/io/proto/guzzle.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto2";
+
+package openhtf;
+
+// Indicates the type of data stored in the payload of a
+// TestRunEnvelope object.  deflate is used for compression.
+enum PayloadType {
+  COMPRESSED_TEST_RUN = 1;
+  COMPRESSED_VARZ = 2;
+  COMPRESSED_LOGFILE = 3;
```

Comment:
This file is going to change again in the next few days, but I'd like to get it in to be able to continue my own work. I'll import that file once it's ready